apps/editorial_features/components/eoy/client.coffee

```diff
@@ -41,7 +41,6 @@ module.exports.EoyView = class EoyView extends Backbone.View
         watchScrolling()
     $(window).resize =>
       @videoControls()
-      @setupCarousel()
```

Comment:
I was noticing that the carousel height is set on initialize at 600 and stays there on resize -- were you able to get this responsive?